relay-server/src/metrics.rs

```diff
@@ -469,6 +475,8 @@ impl CounterMetric for RelayCounters {
             RelayCounters::ProjectCacheMiss => "project_cache.miss",
             RelayCounters::ServerStarting => "server.starting",
             #[cfg(feature = "processing")]
+            RelayCounters::ProcessingEventCorrupted => "processing.event.corrupted",
```

Comment:
_nit_: I would not move this under the `processing` namespace, since it's not a metric about processing. It just happens to be emitted there. Maybe we can have something like `events.corrupted`?